ScanCommand/src/main/java/io/ballerina/scan/internal/ReporterImpl.java

```diff
@@ -72,18 +74,22 @@ private Issue createIssue(Document reportedDocument, Location location, Rule rul
         String moduleName = module.moduleName().toString();
         Path issuesFilePath = module.project().documentPath(reportedDocument.documentId())
                 .orElse(Path.of(documentName));
-
         String fullyQualifiedRuleId = rule.id();
         String[] parts = fullyQualifiedRuleId.split(":");
+
         Source source;
         if (parts[0].equals(BALLERINA_RULE_PREFIX + rule.numericId())) {
             source = Source.BUILT_IN;
         } else {
             String reportedSource = parts[0];
-            String pluginOrg = reportedSource.split(PATH_SEPARATOR)[0];
-            source = pluginOrg.equals(BALLERINA_ORG) ? Source.BUILT_IN : Source.EXTERNAL;
+            String pluginOrg = reportedSource.split(PACKAGE_ORG_NAME_SEPARATOR)[0];
```

Comment:
Is `reportedSource` the correct name here? Reads more like the source for which the issue was reported.